pkg/osm/annotations.go

```diff
@@ -0,0 +1,16 @@
+package osm
+
+// SupportedAnnotations maintains a mapping of OSM version to supported annotations.
+var SupportedIngress = map[ControllerVersion][]Annotation{
+	"v0.5": {},
```

Comment:
still compiling these by prowling the release page

Reply:
@johnsonshi you could use a script like this:
```bash
for x in $(seq 5 9); do git checkout "upstream/release-v0.${x}"; echo "\n----\nv0.$x"; grep -Iir 'Annotation' $(find . -type f -name '*.go' | grep -v _test.go) | grep " = " | grep 'openservicemesh.io/'; done; git checkout main
```